pkg/operator/k8sutil/k8sutil.go

```diff
@@ -94,7 +105,12 @@ func Hash(s string) string {
 // Do **NOT** edit this function in a way that would change its output as it needs to
 // provide consistent mappings from string to hash across versions of rook.
 func TruncateNodeName(format, nodeName string) string {
-	if len(nodeName)+len(fmt.Sprintf(format, "")) > validation.DNS1035LabelMaxLength {
+	return truncateNodeName(format, nodeName, validation.DNS1035LabelMaxLength)
+}
+
+// truncateNodeName takes the max length desired for a string and hashes the value if needed to shorten it.
+func truncateNodeName(format, nodeName string, maxLength int) string {
+	if len(nodeName)+len(fmt.Sprintf(format, "")) > maxLength {
 		hashed := Hash(nodeName)
 		logger.Infof("format and nodeName longer than %d chars, nodeName %s will be %s", validation.DNS1035LabelMaxLength, nodeName, hashed)
```

Comment:
Should the log info message be updated too with `maxLength` instead of `validation.DNS1035LabelMaxLength`?